PathPlanning/PotentialFieldPlanning/potential_field_planning.py

```diff
@@ -128,6 +131,19 @@ def potential_field_planning(sx, sy, gx, gy, ox, oy, reso, rr):
         rx.append(xp)
         ry.append(yp)
 
+        if ((None, None) not in previous_id and
```

Comment:
I think this if evaluation should be a boolean function like "is_oscilication_detected" for readability.

Reply:
good point, done